src/client.coffee

```diff
@@ -213,14 +188,15 @@ class SlackClient
           event.user = fetched.user
           return event
 
-        else if event.bot_id
+        else if fetched.bot
           # bot_id exists on all messages with subtype bot_message
           # these messages only have a user property if sent from a bot user (xoxb token). therefore
           # the above assignment will not happen for all messages from custom integrations or apps without a bot user
-          return Promise.promisify(@findBotUser, { context: @ })(event.bot_id).then((res) =>
-            event.user = res
-            return event
-          )
+          if (fetched.bot.user_id)
```

Comment:
i think you need an `else return event` to make sure the promise chain doesn't break.